tests/providers/amazon/aws/operators/test_ecs_system.py

```diff
@@ -0,0 +1,99 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import pytest
+
+from tests.test_utils.amazon_system_helpers import AWS_DAG_FOLDER, AmazonSystemTest
+
+
+@pytest.mark.system("amazon")
+class ECSSystemTest(AmazonSystemTest):
+    """
+    ECS System Test to run and test example ECS dags
+
+    Required variables.env file content (from your account):
+        # Auto-export all variables
+        set -a
+
+        # aws parameters
+        REGION_NAME="eu-west-1"
+        REGISTRY_ID="123456789012"
+        IMAGE="alpine:3.9"
+        SUBNET_ID="subnet-068e9654a3c357a"
+        SECURITY_GROUP_ID="sg-054dc69874a651"
+        EXECUTION_ROLE_ARN="arn:aws:iam::123456789012:role/FooBarRole"
+
+        # remove all created/existing resources flag
+        # comment out to keep resources or use empty string
+        # REMOVE_RESOURCES="True"
+    """
+
+    # should be same as in the example dag
+    aws_conn_id = "aws_ecs"
+    cluster = "c"
+    task_definition = "hello-world"
+    container = "hello-world-container"
+    awslogs_group = "/ecs/hello-world"
+    awslogs_stream_prefix = "prefix_b"  # only prefix without container name
+
+    def setUp(self):
```

Comment:
```suggestion
    @classmethod
    def setup_class(cli):
```
Small NIT: maybe use `setup_class` is more appropriate here, in case in the future we would have more than one dags run or other task cases. Same to `teardown`.

Reply:
@xinbinhuang actually I don't understand what you mean

Reply:
`set_up` and `tear_down` runs for every test method within the class, so if we have 3 tests (i.e. adding another 2 example dags to ECS) in the future. The `set_up` logic will run 3 times in total. 

On the other hand, `setup_class`  will just apply once so all tests can reutilize the same ECS cluster.

Reply:
is it `setup_class` or `setUpClass`? see [docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass)

Reply:
Either would work, the latter is recognized by Unit Test only, and both are recognized by [`pytest`](https://docs.pytest.org/en/latest/xunit_setup.html).

Reply:
done, can you check it please?

Reply:
Nice! Thank you so much

Reply:
thank you for your review